docs/integrations-in-rancher/rancher-extensions.md

```diff
@@ -49,6 +49,30 @@ In v2.7.0, the built-in extensions will not be displayed under the **Available**
 
     ![Reload button](/img/reload-button.png)
 
+### Importing and Installing Extensions in an Air-Gapped Environment
+
+1. Find the address of the container image you want to import as an extension.
```

Comment:
Rancher provides some extensions such as Kubewarden and elemental and we need to tell the user about this. The extensions are available in this container image https://hub.docker.com/r/rancher/ui-plugin-catalog/tags

The latest available tag is recommended to be used and imported.